### ISSUE TITLE
Unable to access org Integrations page

### ISSUE THREAD

**Issue by attributeerror** [getsentry/sentry#66648]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Log into organisation
2. Navigate to organisation settings -> Custom Integrations

### Expected Result

Able to configure custom integrations

### Actual Result

This is the error I receive in the console whenever I try to access this page:
![Screenshot 2024-02-28 215829](https://github.com/getsentry/sentry/assets/12194893/c429ea5a-0b1d-44d4-aabc-16afcab5f675)

I am able to access it other systems, including mobile, but not my personal laptop. I have tried disabling and blockers, anti-virus, using a different browser, clearing my cache, using incognito/private mode.. all to no avail.

### Product Area

Settings - Integrations

### Link

https://amethyst-it-solutions-ltd.sentry.io/settings/developer-settings/

### DSN

https://b9e2b481a91d4b0880396f1814dca341@o4505392506404864.ingest.us.sentry.io/4505392510271488

**Comment by leedongwei** [comment]
I cannot replicate the issue. If it's working on mobile/etc for you, but not your laptop, then it is probably an issue on your end. Can you try it using another browser on your laptop?

It is likely that you have a browser extension that's overwriting the array prototype functions.

**Comment by attributeerror** [comment]
It's definitely something on my PC as I was just able to access the Custom Integrations page as my PC started, but as apps loaded and I refreshed the page it stopped. I tried accessing the page in incognito & guest mode but to no avail, so it's not an extension. I've only recently re-installed Windows so I have absolutely no idea what could cause this.

**Comment by leedongwei** [comment]
Is this happening in Microsoft Edge on a Windows machine?

**Comment by attributeerror** [comment]
> Is this happening in Microsoft Edge on a Windows machine?

No, Google Chrome on a Windows machine... I have tried Edge though to no avail.

**Comment by leedongwei** [comment]
@attributeerror In your browser console, can you paste in `Array.prototype.filter` and see what's the output? Is your machine managed by your workplace? It seems likely to be something odd on your side as we cannot replicate the error at all.

**Comment by attributeerror** [comment]
@leedongwei Sorry for the delayed response, this is the output of running `Array.prototype.filter` in the console. The machine that I'm experiencing the issue on is **not** managed by my workplace. Definitely is something on my side, I just can't figure it out...
![image](https://github.com/getsentry/sentry/assets/12194893/afebbe0b-5b51-4e18-9efe-398622b3e178)
